plugins/pcl/filters/PCLBlock.cpp

```diff
@@ -129,7 +129,7 @@ PointBufferSet PCLBlock::run(PointBufferPtr input)
     if (cloud_f->points.empty())
     {
         log()->get(LogLevel::Debug2) << "Filtered cloud has no points!" << std::endl;
-        return pbSet;
+        return viewSet;
     }
 
     pclsupport::PCDtoPDAL(*cloud_f, *output, buffer_bounds);
```

Comment:
`Error	474	error C2664: 'void pdal::pclsupport::PCDtoPDAL<T>(CloudT &,pdal::PointViewPtr,const pdal::BOX3D &)' : cannot convert parameter 2 from 'pdal::PointView' to 'pdal::PointViewPtr'`